test-apps/display-test-app/package.json

```diff
@@ -20,7 +20,7 @@
     "copy:assets": "cpx \"./node_modules/@bentley/icons-generic-webfont/dist/**/*\" ./build",
     "docs": "",
     "lint": "eslint -f visualstudio --config package.json --no-eslintrc ./src/**/*.ts 1>&2",
-    "start": "run-p start:webserver start:electron",
+    "start": "cross-env NODE_ENV=development run-p start:webserver start:electron",
```

Comment:
@pmconne @kabentley changed this because the way it currently is setup started the webserver even though it wasn't being used.  Is that intended or by accident?

Reply:
I don't know. I vaguely recall that it used to start in development mode by default, and that it changed around the time @kabentley made his "node integration" changes such that we now have to manually set NODE_ENV before running.

Reply:
what i changed was the default in electron-manager (it used to default to development, and i made it default to production. so you didn't have to set NOD_ENV in production. Most users are not test apps.) 

Does your change test whether NODE_ENV === development and if so not start webserver?

Reply:
sorry, the change to default was in display-test-app, not electron-manager. I did that because people were copying this code for their real apps.

Reply:
Ah okay, so is it okay with directly set it in the `start` command here so the default when running it is using the webserver?